src/Server/Validation/HttpServerValidator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace K911\Swoole\Server\Validation;
+
+use K911\Swoole\Server\Config\Socket;
+use K911\Swoole\Server\Validation\Exception\AlreadyAttached;
+use K911\Swoole\Server\Validation\Exception\InvalidServerPort;
+use K911\Swoole\Server\Validation\Exception\NoServerAttached;
+use K911\Swoole\Server\Validation\Exception\PortAlreadyRegistered;
+use Swoole\Http\Server;
+
+class HttpServerValidator
```

Comment:
The same as HttpClientValidator, please drop this class and use private methods. I understand that it may look better for now, but I'm afraid it means that this class needs to be maintained in the future, because it's public, and we cannot afford this ATM. Also, I think it would be better to split some parts of the bundle to lighter components.